docs/insomnia/chaining-requests.md

```diff
@@ -5,57 +5,40 @@ category: "Requests and Responses"
 
-First, let's take a look at some examples that chaining may be useful for.
+* use a token from a login response as an Authorization header
+* use the ID from POST `/items` in PUT `/items/{id}`
+* use an entire body from one response in another request
 
-## Common Uses
+## Select Response Tag
 
-To help explain request chaining, here are some common use cases:
+In an empty URL bar or following a URL symbol such as `/` or `=`, either press CTRL+Space to open the autocomplete dropdown or start typing "response".
 
-* use token from login response as an Authorization header
-* use id from POST `/item in PUT /items/<id>`
-* use entire body from one response to upload to another
+There are three response options:
 
-There are many other ways to use chaining, but those are the most commonly used examples.
+* **Response - Body Attribute**
+  * Works on JSON and XML responses, pulling attributes from a response body.
 
-## Chaining Basics
+    Use either JSONPath or XPath queries (depending on the Content-Type) to extract specific attributes from a response body. For example, a login request may respond with a JSON object that looks like `{"auth_token": "secret-token"}`. This allows you to use a JSONPath filter of `$.auth_token` to reference the token itself in a subsequent request.
+* **Response - Raw Body**
+  * References the entire body contents of a request. This can be useful if you download content in one request that needs to be uploaded in another.
+* **Response - Header**
+  * Pulls specific header values out of a response.
 
-To get started, select one of three Response tags from the autocomplete dropdown. You can invoke the dropdown be pressing CTRL+Space or by typing starting characters of the tag (respo...). The response tag can be used anywhere that environment variables are supported (URL, headers, request body, environments, etc).
+The Response tag can be used anywhere [Environment Variables](https://docs.insomnia.rest/insomnia/environment-variables) are supported.
 
-Reference the id property of a created resource from another request. Then, click the tag to select the request and specify the desired parameters. This will open a dialog where you can configure the tag.
+## Configure a Tag
 
-{:.alert .alert-primary}
-**Note**: Use this tag inside an Environment Variable to prevent duplicated effort
+Click on the tag in the URL bar to configure it. An **Edit Tag** modal will open. Select options from the first four fields:
 
-Now, you can configure the tag parameters and see what the result will be in the Live Preview section of the dialog.
+* **Function to Perform**: select the function to perform or value to reference.
+* **Attribute**: select the value to perform the action on.
+* **Request**: select the request to apply the configure onto.
+* **Trigger Behavior**: select the application behavior that populates the tag. Click on the cog button to select either Static Value or Environment Variable.
```

Comment:
Trigger behavior specifies what should happen when data from the selected request is needed. Don't this description is applicable?

![image](https://user-images.githubusercontent.com/4312346/146460625-58ef9642-6fa9-47ef-85bd-060bfbc91f5d.png)